src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/elbow/ElbowMoveByJoysticks.java

```diff
@@ -22,7 +22,7 @@ public void execute() {
 		double power = joystick.getAxisValue(SmartJoystick.Axis.RIGHT_Y) * 1;
 		SmartDashboard.putNumber("volt elbow", power * Battery.getInstance().getCurrentVoltage() + Elbow.getStaticFeedForward(Extender.getInstance().getLength(), elbow.getAngleRadians()));
 		SmartDashboard.putNumber("elbow vel", elbow.getVelocity());
-		elbow.setMotorVoltage(power * Battery.getInstance().getCurrentVoltage() + Elbow.getStaticFeedForward(Extender.getInstance().getLength(), elbow.getAngleRadians()));
+		elbow.setMotorVoltage(power * Battery.getInstance().getCurrentVoltage() + Elbow.getDynamicFeedForward(Extender.getInstance().getLength(), elbow.getAngleRadians()));
```

Comment:
dynamic feed forward should not be used in move by joystick as there isnt a set velocity target

Reply:
changed to static